src/Controls/src/Core/FontRegistrar.cs

```diff
@@ -19,6 +19,52 @@ public static void Register(ExportFontAttribute fontAttribute, Assembly assembly
 				EmbeddedFonts[fontAttribute.Alias] = (fontAttribute, assembly);
 		}
 
+		static bool TryLoadFont(string cacheKey, Stream fontStream)
+		{
+			var type = Registrar.Registered.GetHandlerType(typeof(EmbeddedFont));
+			var fontHandler = (IEmbeddedFontLoader)Activator.CreateInstance(type);
+
+			var result = fontHandler.LoadFont(new EmbeddedFont { ResourceStream = fontStream });
+
+			fontLookupCache[cacheKey] = result;
+
+			return result.success;
+		}
+
+		public static bool RegisterNative(string fontName, string alias = null)
+		{
+			var cacheKey = alias ?? fontName;
+
+#if __IOS__ || IOS
+			var mainBundlePath = NSBundle.MainBundle.BundlePath;
+			var fontBundlePath = Path.Combine(mainBundlePath, fontName);
+
+			if (!File.Exists(fontBundlePath))
+				fontBundlePath = Path.Combine(mainBundlePath, "Resources", fontName);
+
+			if (!File.Exists(fontBundlePath))
+				fontBundlePath = Path.Combine(mainBundlePath, "Fonts", fontName);
+
+			if (!File.Exists(fontBundlePath))
+				fontBundlePath = Path.Combine(mainBundlePath, "Resources", "Fonts", fontName);
+
+			if (!File.Exists(fontBundlePath))
+				return false;
+
+			using (var fontStream = File.OpenRead(fontBundlePath))
+			{
+				return TryLoadFont(cacheKey, fontStream);
+			}
+#elif __ANDROID__ || ANDROID
+			var assets = Android.App.Application.Context.Assets;
+			using (var fontStream = assets.Open(fontName))
```

Comment:
We probably need to do the paths better. On iOS we got a few locations, Android is just at the root. I personally will always use a subfolder. 

Maybe we need to allow for some property when resizing and here to provide a sub-path.